skiko/src/awtTest/kotlin/org/jetbrains/skiko/SkiaLayerTest.kt

```diff
@@ -386,6 +398,10 @@ class SkiaLayerTest {
             window3.toFront()
             delay(1000)
             screenshots.assert(window3.bounds, "window3")
+
+            assertRenderApiFor(window1.layer)
```

Comment:
This is added to the other tests to not miss the case when the test fallbacks to the next API due to not supporting it